indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -119,3 +126,51 @@ def _process_PATCH(self):
             updates['colors'] = colors
         update_session(self.session, updates)
         return jsonify()
+
+
+class RHSessionPersonList(RHManageSessionsBase):
+    """List of persons in the session"""
+
+    def _process(self):
+        session_ids = map(int, request.form.getlist('session_id'))
```

Comment:
Yep, simply this code had been written before I introduced `RHManageSessionsActionsBase` and I forgot to change it.